### PR TITLE
chore(docs): add count(column) documentation

### PR DESCRIPTION
**Pull request by bziobrowski**

### CODE REVIEW

File: docs/reference/function/aggregation.md [hunk 1]
```diff
@@ -76,9 +79,32 @@ SELECT payment_type, count() FROM transactions;
...
+|:------|
+| 95    |
+               
+- Count non-null transaction amounts by payment_type
+
+```questdb-sql 
+SELECT payment_type, avg(amount) FROM transactions;
+```
+
+| cash_or_card | count |
```
Comment (amyshwang) [inline docs/reference/function/aggregation.md:98]: Should this be:
```suggestion
| payment_type | count |
```
`cash_or_card` is used throughout the example section - I can update it but would like to check this needs to be changed indeed.

File: docs/reference/function/aggregation.md [hunk 2]
```diff
@@ -76,9 +79,32 @@ SELECT payment_type, count() FROM transactions;
...
+```
+
+| count |
+|:------|
+| 95    |
+               
+- Count non-null transaction amounts by payment_type
+
+```questdb-sql 
+SELECT payment_type, avg(amount) FROM transactions;
```
Comment (amyshwang) [inline docs/reference/function/aggregation.md:95]: I think it should be:
```suggestion
SELECT payment_type, count(amount) FROM transactions;
```

File: docs/reference/function/aggregation.md [hunk 3]
```diff
@@ -42,11 +42,14 @@ SELECT payment_type, avg(amount) FROM transactions;
 
 ## count
 
-`count()` or `count(*)` - counts rows irrespective of underlying data.
+- `count()` or `count(*)` - counts rows irrespective of underlying data.
+- `count(column_name)` - counts number of non-null values in given column.
```
Comment (amyshwang) [inline docs/reference/function/aggregation.md:46]:
```suggestion
- `count(column_name)` - counts the number of non-null values in a given column. 
```

File: docs/reference/function/aggregation.md [hunk 4]
```diff
@@ -42,11 +42,14 @@ SELECT payment_type, avg(amount) FROM transactions;
 
 ## count
 
-`count()` or `count(*)` - counts rows irrespective of underlying data.
+- `count()` or `count(*)` - counts rows irrespective of underlying data.
```
Comment (amyshwang) [inline docs/reference/function/aggregation.md:45]:
```suggestion
- `count()` or `count(*)` - counts the number of rows irrespective of underlying data.
```

File: docs/reference/function/aggregation.md [hunk 5]
```diff
@@ -42,11 +42,14 @@ SELECT payment_type, avg(amount) FROM transactions;
...
+- `count(column_name)` - counts number of non-null values in given column. 
 
 **Arguments:**
 
-- `count` does not require arguments.
+- `count()` does not require arguments.
+- `count(column_name)` - supports following data types: `double`, `float`, `integer`, `character`, `short`,
```
Comment (amyshwang) [inline docs/reference/function/aggregation.md:51]:
```suggestion
- `count(column_name)` - supports the following data types: `double`, `float`, `integer`, `character`, `short`,
```

**Review by amyshwang — COMMENTED** [review]
Some editorial suggestions and questions about the examples used...

File: docs/reference/function/aggregation.md (hunk 1 above)
Comment (bziobrowski) [reply docs/reference/function/aggregation.md:98]: To be honest cash_ord_card is not a realistic column name/header for any of the quries in aggregation.md :)

File: docs/reference/function/aggregation.md (hunk 5 above)
Comment (bziobrowski) [reply docs/reference/function/aggregation.md]: Latest commit should fix all issues you've raised .

**Review by amyshwang — APPROVED** [review]

**Pull request merged**